TaskSharper.Calender.WPF/ViewModels/CalendarWeekViewModel.cs

```diff
@@ -64,16 +94,19 @@ public Task GetCalelndarEvents()
             //TODO:: Refactor this to make use of observable dictionary
```

Comment:
Spelling error in method name (Should be GetCalendarEvents()).

Reply:
Fixed